postgres/queries/organisations.sql

```diff
@@ -20,33 +21,34 @@ FROM
     organisations;
 
 -- name: CreateOrganisation :one
-INSERT INTO organisations (name, parent_id)
-    VALUES ($1, $2)
+INSERT INTO
+    organisations (name, parent_id)
+VALUES
+    ($1, $2)
 RETURNING
     *;
 
-
 -- name: UpdateOrganisation :one
-UPDATE
-    organisations
+UPDATE organisations
```

Comment:
Dit lijkt me de omgekeerde formatting aanpassing van wat je op andere plaatsen doet in deze PR? 😛

Reply:
Tja, mijn formatter doet dit vanzelf (prettier-sql)

Reply:
Kan je die aanpassen om dezelfde formatting te gebruiken als [pgFormatter](https://github.com/darold/pgFormatter) (zonder opties)? Ik heb het net eens uitgetest en die plaatst de tabel wel altijd op een aparte lijn (zoals het hiervoor ook was). De bedoeling van autoformatting is dat het bij iedereen consistent op dezelfde manier geformat wordt zodat er niet continu conflicterende formatting aanpassingen zouden zijn. Dat gaat niet als er iemand editor-specifieke tools gebruikt ipv algemene tools die iedereen in zijn eigen editor kan gebruiken 😛

Reply:
`sqlfmt` doet dat ook op 1 lijn, ik weet niet waarom want ik vind persoonlijk een newline wel duidelijker in die statements (zeker als ze lang worden), maar cross-platform consistentie is nog belangrijker 😛